README.md

```diff
@@ -1,65 +1,92 @@
 # MaxCompute Go Driver
 
-This repo contains a Go driver for MaxCompute(ODPS) by AliCloud. If your Go projects involve with talking to MaxCompute server, you are welcome to try it out and give us some feedback. Note this implementation contains a database abstraction layer `database/sql` compatible with Go standard library using native HTTP API. As you can see from version number, this repo is under fast iteration, please create issue if you find anything missing.
+[MaxCompute](https://www.alibabacloud.com/product/maxcompute), also known as ODPS, is a distributed storage service and SQL engine provided by [Alibaba Cloud](www.alibabacloud.com/). This repository contains a Go [SQLdriver](https://github.com/golang/go/wiki/SQLDrivers) of MaxCompute.  If you are going to write a Go program that calls the standard library `database/sql` to access MaxCompute databases, you could use this driver.
 
-## Features
-- Light weight and fast
-- Native Go implementation, compatible with Go interface `database/sql`
-- Connection through [HTTP API](http://repo.aliyun.com/api-doc/)
-- Supports large queries via MaxCompute tunnel server
+This project is in its early stage. Your issues and pull requests are very welcome!
+
+
+## What This Is and Isn't
+
+This project is a driver that helps Go's standard database API talking to MaxCompute server. It has the following features:
+ 
+- In pure Go. Not a wrapper of any C/C++ library.
+- Connect to MaxCompute through its [HTTP interface](http://repo.aliyun.com/api-doc/).
+- Improve I/O throughput using MaxCompute's [tunnel service](https://www.alibabacloud.com/help/doc-detail/27833.htm).
```

Comment:
Improved ?

Reply:
We used "Connect to" in the previous sentence, so here it should be "Improve".